packages/internals/src/errorReporting.ts

```diff
@@ -80,7 +80,7 @@ async function request(query: string, variables: any): Promise<any> {
       if (!response.ok) {
         throw new Error(`Error during request: ${response.status} ${response.statusText} - Query: ${query}`)
       }
-      return response.json()
+      return response.json() as any
```

Comment:
**note**

In the new version `json()` return `unknown` and not `any` so this cast was necessary to avoid type error in the next `.then`